### DISCUSSION TITLE
go2rtc not working on V2 cam

### DISCUSSION THREAD

**Discussion by siriuslooker** [gtxaspec/wz_mini_hacks#576] [Q&A] [answered]
Running the alternative method of flashing new uboot image. Running stock firmware 4.9.8.1002.
wz_mini boots up, ssh, web gui and go2rtc web gui all work fine.

The minute I connect to one of the go2rtc published streams, I see in the go2rtc logs that the device /dev/video1 or /dev/video2 "resource is busy" and I get either a failure to connect to the rtsp stream or a black screen on the webrtc stream.

**Comment by gtxaspec** [comment, marked as answer]
https://github.com/gtxaspec/wz_mini_hacks/wiki/Video-Streaming

Change the video device in the go2rtc config.

**Reply by siriuslooker** [reply]
Thanks, I totally didn't see that part of the docs.
